magenta-lib/src/main/scala/magenta/deployment_type/AutoScaling.scala

```diff
@@ -123,6 +123,7 @@ object AutoScaling  extends DeploymentType with S3AclParams {
         )
       List(
         S3Upload(
+          target.region,
```

Comment:
That's an interesting philosophical question. This would mean that you get multiple uploads at present. The config lets you avoid this as you can override the region - but you are right, it's probably not helpful.